wondrous-app/graphql/queries/org.ts

```diff
@@ -64,6 +64,9 @@ export const GET_ORG_USERS = gql`
         firstName
         lastName
         bio
+        additionalInfo {
+          podCount
```

Comment:
this is actually the wrong data, this is total pod count across organizations. let's just remove the pod count field